tests/e2e_all/scripts/common/apkam_setup.sh

```diff
@@ -72,7 +72,7 @@ enroll() {
     logInfo "keys file HAS been created at $keysFileName"
     return 0
   else
-    logError "keys file has NOT been created at $keysFileName"
+    logErrorAndReport "keys file has NOT been created at $keysFileName"
```

Comment:
Intentionally disabled the `AndReport` in many places since many parts of the program are being asynchronously executed in parallel and it will mess up the order of the report.